indy_node/server/node.py

```diff
@@ -378,15 +302,8 @@ def executeDomainTxns(self, ppTime, reqs: List[Request], stateRoot,
         :param ppTime: the time at which PRE-PREPARE was sent
         :param req: the client REQUEST
         """
-        req_handler = self.get_req_handler(DOMAIN_LEDGER_ID)
-        return self.commitAndSendReplies(req_handler, ppTime, reqs,
-                                         stateRoot, txnRoot)
-
-    def executeConfigTxns(self, ppTime, reqs: List[Request], stateRoot,
-                          txnRoot) -> List:
-        req_handler = self.get_req_handler(CONFIG_LEDGER_ID)
-        return self.commitAndSendReplies(req_handler, ppTime, reqs,
-                                         stateRoot, txnRoot)
+        return self.default_executer(DOMAIN_LEDGER_ID, ppTime, reqs,
```

Comment:
Why can't we have a generic method in plenum for execution of txns?
Do we need to override `executeDomainTxns`?

Reply:
Because `executeDomainTxns` in plenum does an extra action which is not need for this one